ksa_compliance/ksa_compliance/doctype/zatca_business_settings/zatca_business_settings.py

```diff
@@ -126,6 +123,15 @@ def _sandbox_private_key_path(self) -> str:
                 f.write(key.encode('utf-8'))
         return path
 
+    @property
+    def fatoora_server_url(self) -> str:
+        if self.fatoora_server == "Sandbox":
+            return 'https://gw-fatoora.zatca.gov.sa/e-invoicing/developer-portal/'
+        if self.fatoora_server == "Simulation":
+            return 'https://gw-fatoora.zatca.gov.sa/e-invoicing/simulation/'
+        if self.fatoora_server == "Production":
+            return 'https://gw-fatoora.zatca.gov.sa/e-invoicing/core/'
```

Comment:
We need to throw an exception if we reach the end of the function without matching any server